v0/destinations/drip/data/DripIdentify.json

```diff
@@ -0,0 +1,136 @@
+[
+  {
+    "destKey": "email",
```

Comment:
since we are already extracting email from `identifyResponseBuilder`, we do not need this mapping here.